src/Form/tests/FormAutosuggest.test.jsx

```diff
@@ -1,22 +1,12 @@
 import React from 'react';
-import { mount } from 'enzyme';
-import { act } from 'react-dom/test-utils';
+import PropTypes from 'prop-types';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
```

Comment:
Is it better to move this line to `setupTests.js`? Did you encounter some problems with this?